tools/kmodules/sched_tp/Makefile

```diff
@@ -1,3 +1,31 @@
 obj-m += sched_tp.o
 
 EXTRA_CFLAGS = -I$(src)
+
+VMLINUX_DEPS_H = vmlinux_deps.h
+VMLINUX_H = vmlinux.h
+
+VMLINUX_DEPS_TXT = vmlinux_deps.txt
+VMLINUX_TXT = vmlinux.txt
+
+ifeq ($(wildcard $(KERNEL_SRC)/vmlinux), )
+	VMLINUX = /sys/kernel/btf/vmlinux
```

Comment:
We should probably remove this case, LISA is expected to be installed on a host that is different from the target and everything cross compiled, so this Makefile will be called on the host machine, not the development board.
Therefore we can't use the host's kernel to infer anything about the kernel's target.

Reply:
Updated the changed.